cpp/oneapi/dal/algo/decision_forest/test/batch.cpp

```diff
@@ -170,12 +170,12 @@ class df_batch_test : public te::algo_fixture {
         const auto infer_result = this->infer(desc, model, x_test);
         check_infer_shapes(desc, data, infer_result);
 
-        SECTION("infer accuracy is expected") {
-            for (auto ch : checker_list) {
-                CAPTURE(ch.name);
-                REQUIRE(ch.check(infer_result.get_labels(), y_test) < ch.required_accuracy + eps);
-            }
+        for (auto ch : checker_list) {
+            CAPTURE(ch.name);
+            REQUIRE(ch.check(infer_result.get_labels(), y_test) < ch.required_accuracy + eps);
         }
+        INFO("infer accuracy is expected")
```

Comment:
Info should go before the actual check, and it's better to rephrase like "check if infer accuracy is expected"